alf/examples/ppo_conf.py

```diff
@@ -24,7 +24,14 @@
 
 alf.config('EntropyTargetAlgorithm', initial_alpha=1.)
 
-alf.config('PPOLoss', entropy_regularization=None, normalize_advantages=True)
+# use proper positive value for stablization, especially for continuous control
+log_prob_clipping = 100
+
+alf.config(
+    'PPOLoss',
+    entropy_regularization=None,
+    normalize_advantages=True,
+    log_prob_clipping=log_prob_clipping)
```

Comment:
This may have impact on all configurations that includes `ppo_conf.py`, may need to update those as well.